packages/forms/stories/index.js

```diff
@@ -114,6 +114,82 @@ decoratedStories.add('Multiple actions', () => {
 	);
 });
 
+decoratedStories.add('Datalist', () => {
+	function fetchItems(title) {
+		console.log('fetchItems', title);
+		return [
```

Comment:
Can you change the content to show in the story that the list is different from the enum in jsonSchema ?